src/tokenizer/main.py

```diff
@@ -150,6 +213,30 @@ def val(t, quote_word=False):
     else:
         to_text = lambda t: t.txt
 
+    if args.convert_measurements:
+        options["convert_measurements"] = True
+
+    if args.with_annotation:
+        options["with_annotation"] = True
+
+    if args.coalesce_percent:
+        options["coalesce_percent"] = True
+
+    if args.replace_composite_glyphs:
+        options["replace_composite_glyphs"] = True
+
+    if args.replace_html_escapes:
+        options["replace_html_escapes"] = True
+
+    if args.convert_numbers:
+        options["convert_numbers"] = True
+
+    if args.onesentperline:
```

Comment:
Change this to underscores (and I actually think the code doesn't work without them as it is).

Reply:
Done!